colossalai/communication/__init__.py

```diff
@@ -14,4 +16,6 @@
     'send_backward_recv_backward', 'send_backward_recv_forward',
     'send_forward_recv_backward', 'recv_backward', 'recv_forward',
     'ring_forward', 'send_tensor_meta', 'recv_tensor_meta',
+    'CCPROF', 'cc_wait_aync', 'cc_all_reduce', 'cc_reduce_scatter',
+    'cc_all_gather', 'cc_broadcast', 'cc_reduce'
```

Comment:
cc_ means colossal?

Reply:
cc = colossal communication. Anyway, it can be changed.